[2023-01-17T23:25:12.449+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.store_user manual__2023-01-17T23:25:06.280408+00:00 [queued]>
[2023-01-17T23:25:12.455+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.store_user manual__2023-01-17T23:25:06.280408+00:00 [queued]>
[2023-01-17T23:25:12.455+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-01-17T23:25:12.455+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-01-17T23:25:12.455+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-01-17T23:25:12.464+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): store_user> on 2023-01-17 23:25:06.280408+00:00
[2023-01-17T23:25:12.469+0000] {standard_task_runner.py:55} INFO - Started process 2107 to run task
[2023-01-17T23:25:12.471+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'store_user', 'manual__2023-01-17T23:25:06.280408+00:00', '--job-id', '200', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp2wqmosrr']
[2023-01-17T23:25:12.472+0000] {standard_task_runner.py:83} INFO - Job 200: Subtask store_user
[2023-01-17T23:25:12.508+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.store_user manual__2023-01-17T23:25:06.280408+00:00 [running]> on host 274e1da73cdf
[2023-01-17T23:25:12.551+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=store_user
AIRFLOW_CTX_EXECUTION_DATE=2023-01-17T23:25:06.280408+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-17T23:25:06.280408+00:00
[2023-01-17T23:25:12.552+0000] {postgres.py:129} INFO - Running copy expert: COPY users FROM stdin WITH DELIMITER as ',', filename: /tmp/processed_user.csv
[2023-01-17T23:25:12.557+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-01-17T23:25:12.561+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-01-17T23:25:12.567+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=store_user, execution_date=20230117T232506, start_date=20230117T232512, end_date=20230117T232512
[2023-01-17T23:25:12.602+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-01-17T23:25:12.632+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
